veracruz-server/src/common.rs

```diff
@@ -104,21 +103,12 @@ pub trait VeracruzServer {
     where
         Self: Sized;
 
-    fn plaintext_data(&mut self, _data: Vec<u8>) -> VeracruzServerResult<Option<Vec<u8>>> {
-        // this function is not strictly needed, should we remove at some point?
-        unimplemented!();
-    }
-
     fn new_tls_session(&mut self) -> VeracruzServerResult<u32>;
 
-    fn close_tls_session(&mut self, session_id: u32) -> VeracruzServerResult<()>;
-
     // The first bool indicates if the enclave is active, and the second vec contains the response
     fn tls_data(
         &mut self,
         session_id: u32,
         input: Vec<u8>,
     ) -> VeracruzServerResult<(bool, Option<Vec<Vec<u8>>>)>;
-
-    fn shutdown_isolate(&mut self) -> Result<(), Box<dyn Error>>;
```

Comment:
Why was this moved outside of the `VeracruzServer` trait? Shouldn't `shutdown_isolate()` be implemented for every target?

Reply:
It's implemented, but it's not in the API any more because the shutdown is initated by dropping the struct: https://github.com/veracruz-project/veracruz/blob/2682973574a1424352f8905a6b74b470a36f4145/veracruz-server/src/server.rs#L61